src/app/router/connectorRoutes.tsx

```diff
@@ -30,12 +33,173 @@ import raspiblitz from "/static/assets/icons/raspiblitz.png";
+    logo?: string;
+  }[];
+}
+
 const galoyPaths: { [key: string]: keyof typeof galoyUrls } = {
   bitcoinBeach: "galoy-bitcoin-beach",
   bitcoinJungle: "galoy-bitcoin-jungle",
 };
 
-function getConnectorRoutes() {
+const umbrelConnectorRoutes = [
+  {
+    path: "lnd-connect",
+    element: <ConnectUmbrel />,
+    title: i18n.t("translation:choose_connector.lightning_node.title"),
+    logo: lightning_node,
+  },
+  {
+    path: "lnc",
+    element: <ConnectLnc />,
+    title: i18n.t("translation:choose_connector.lnc.title"),
+    logo: lightning_terminal,
+  },
+  {
+    path: "commando",
+    element: <ConnectCommando />,
+    title: i18n.t("translation:choose_connector.commando.title"),
+    logo: core_ln,
+  },
+  {
+    path: "lnbits",
+    element: <ConnectLnbits />,
+    title: i18n.t("translation:choose_connector.lnbits.title"),
+    logo: lnbits,
+  },
+];
+
+const citadelConnectorRoutes = [
+  {
+    path: "citadel",
+    element: <ConnectCitadel />,
+    title: i18n.t("translation:choose_connector.citadel.title"),
+    logo: citadel,
+  },
+  {
+    path: "lnc",
+    element: <ConnectLnc />,
+    title: i18n.t("translation:choose_connector.lnc.title"),
+    logo: lightning_terminal,
+  },
+  {
+    path: "commando",
+    element: <ConnectCommando />,
+    title: i18n.t("translation:choose_connector.commando.title"),
+    logo: core_ln,
+  },
+  {
+    path: "lnbits",
+    element: <ConnectLnbits />,
+    title: i18n.t("translation:choose_connector.lnbits.title"),
+    logo: lnbits,
+  },
+];
+
+const raspiblitzConnectorRoutes = [
+  {
+    path: "lnd",
+    element: <ConnectRaspiBlitz />,
+    title: i18n.t("translation:choose_connector.lnd.title"),
+    logo: lnd,
+  },
+  {
+    path: "lnc",
+    element: <ConnectLnc />,
+    title: i18n.t("translation:choose_connector.lnc.title"),
+    logo: lightning_terminal,
+  },
+  {
+    path: "commando",
+    element: <ConnectCommando />,
+    title: i18n.t("translation:choose_connector.commando.title"),
+    logo: core_ln,
+  },
+  {
+    path: "lnbits",
+    element: <ConnectLnbits />,
+    title: i18n.t("translation:choose_connector.lnbits.title"),
+    logo: lnbits,
+  },
+];
+
+const myNodeConnectorRoutes = [
+  {
+    path: "lnd",
+    element: <ConnectMyNode />,
+    title: i18n.t("translation:choose_connector.lnd.title"),
+    logo: lnd,
+  },
+  {
+    path: "lnc",
+    element: <ConnectLnc />,
+    title: i18n.t("translation:choose_connector.lnc.title"),
+    logo: lightning_terminal,
+  },
+  {
+    path: "commando",
+    element: <ConnectCommando />,
+    title: i18n.t("translation:choose_connector.commando.title"),
+    logo: core_ln,
+  },
+  {
+    path: "lnbits",
+    element: <ConnectLnbits />,
+    title: i18n.t("translation:choose_connector.lnbits.title"),
+    logo: lnbits,
+  },
+];
+
+const start9ConnectorRoutes = [
+  {
+    path: "lnd",
+    element: <ConnectStart9 />,
+    title: i18n.t("translation:choose_connector.lnd.title"),
+    logo: lnd,
```

Comment:
The definition of different connectors is repeated quite some times. Could we define the connectors and then only assign those values to the different connector routes so we can maintain them in one place?

Reply:
I DRYied this up and created a map object to only define each connector object once.